README.md

```diff
@@ -62,7 +63,29 @@ internet.
 [kube-providers]: https://skupper.io/start/kubernetes.html
 [install-kubectl]: https://kubernetes.io/docs/tasks/tools/install-kubectl/
 
-## Step 1: Access your Kubernetes cluster
+## Step 1: Install the Skupper command-line tool
+
+This example uses the Skupper command-line tool to create Skupper
+resources.  You need to install the `skupper` command only once
+for each development environment.
+
+On Linux or Mac, you can use the install script (inspect it
```

Comment:
this is unfortunate, will users think they can do this example on Mac?

Reply:
Good point. Would it be worth adding a pre-requisite item to use a linux machine?